packages/storefront-sdk-plugins/commerce-queries/src/utils/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const dataFetchingMethods = [
+	'content',
+	'navigation',
+	'products',
+	'productCollections',
+	'productCollectionEntries',
+	'query',
+	'spaceProperties'
+] as const;
```

Comment:
Is there a reason we're using this as an array of values vs a string literal type? 


Eg this over something like this: 


```ts
export type dataFetchingMethods = 'content' | 'navigation' | 'products' | 'productCollections' | 'productCollectionEntries' | 'query' | 'spaceProperties'
```

Reply:
Refactored in https://github.com/getnacelle/nacelle-js/pull/323/commits/02b418f144290bf999f9abd3355f2703094870dd.